example-tree.json

```diff
@@ -1,65 +1,74 @@
-        "children": [
-            1,
-            2
-        ]
-    },
-    {
-        "id": 1,
-        "iconName": "mindfulness",
-        "title": "Mindfulness I",
-        "level": 1,
-        "goal": "Mindfulness 2mins",
-        "frequency": "DAILY",
-        "timelimit": "1x Week",
-        "xp": 50,
+        "title": "babble",
+        "level": 2,
+        "goal": [
+            "annihalate infants",
+            "buy food"
+        ],
+        "frequency": 2,
+        "interval": "day",
+        "timelimit": "1 WEEK",
+        "xp": 1500,
+        "icon": "apple",
         "category": "health",
-        "type": "skill",
-        "requires": [],
-        "children": []
+        "requires": [
+            "629619a3be6df3d69aa702e5"
+        ],
+        "children": [],
+        "type": "skills",
+        "id": "62961972be6df3d69aa702e2"
     },
     {
-        "id": 2,
-        "iconName": "journaling",
-        "title": "Journaling I",
+        "title": "gaggle",
         "level": 1,
-        "goal": "Journal",
-        "frequency": "DAILY",
-        "timelimit": "1x Week",
-        "xp": 100,
-        "category": "skills",
-        "type": "skill",
+        "goal": [
+            "test skills",
+            "make the damn web editor"
+        ],
+        "frequency": 4,
+        "interval": "week",
+        "timelimit": "2 WEEKS",
+        "xp": 1000,
+        "icon": "apple",
+        "category": "fitness",
         "requires": [],
-        "children": [3, 4]
+        "children": [
+            "62961972be6df3d69aa702e2"
+        ],
+        "type": "skills",
+        "id": "629619a3be6df3d69aa702e5",
+        "isRoot": true
     },
     {
-        "id": 3,
-        "title": "Ancient Library",
-        "category": "Learning",
-        "link": "https://drive.google.com/drive/folders/1HGG7si_q8bfmT4eHEH0vcytn9TegRtPy",
-        "type": "item",
+        "title": "foobar",
+        "level": 3,
+        "goal": [
+            "testing goal",
+            "should be gucci"
+        ],
+        "frequency": 3,
+        "interval": "week",
+        "timelimit": "1 WEEKS",
+        "xp": 1200,
+        "icon": "banana",
+        "category": "fitness",
         "requires": [],
-        "children": []
+        "children": [
+            "62968c920e15c4cb06a8283b"
+        ],
+        "type": "skills",
+        "id": "62968af5af70a3169a1a6fc9",
+        "isRoot": true
     },
     {
-        "id": 4,
-        "iconName": "relationships",
-        "title": "Relationships I",
-        "level": 1,
-        "goal": "Find a partner",
-        "frequency": "DAILY",
-        "timelimit": "1x Week",
-        "xp": 100,
-        "category": "skills",
-        "type": "skill",
-        "requires": [],
-        "children": []
+        "name": "item test",
+        "link": "omersabic.xyz",
+        "emoji": "😋",
+        "requires": [
+            "62968af5af70a3169a1a6fc9"
+        ],
+        "type": "items",
+        "id": "62968c920e15c4cb06a8283b"
```

Comment:
Probably don't want to use "real" id's, if these are real. Unless really required.

Same goes for line 69, 61, 58, etc.

Reply:
Sounds good, bit of an oversight but will need to change that. Thanks